sentry/src/main/java/io/sentry/SentryItemType.java

```diff
@@ -6,6 +6,7 @@
 public enum SentryItemType {
   Session("session"),
   Event("event"), // DataCategory.Error
+  User_Report("user_report"), // UserFeedback
```

Comment:
```suggestion
  UserReport("user_report"), // UserFeedback
```

Reply:
H; as we are never using Method_Name

Reply:
Oh sorry, I forgot to put a comment in here. We use `name` instead of `getItemType()` in here.
https://github.com/getsentry/sentry-java/blob/b0bff7b14bc42be8f3ee350ee087bcd8582659b8/sentry/src/main/java/io/sentry/SentryEnvelopeItemHeaderAdapter.java#L34

I changed it now in this PR. Would you prefer another PR for it or are you fine with this in here?

Reply:
mmm true, otherwise the `type` on the json will be wrong :(
a comment is fine, otherwise, it'd require more code/refactoring